wrappers/services.go

```diff
@@ -1414,6 +1454,19 @@ WantedBy={{.SocketsTarget}}
 	return templateOut.Bytes()
 }
 
+func genJournalServiceFile(grp *quota.Group) []byte {
+	buf := bytes.Buffer{}
+	template := `[Unit]
+Description=Journal Service for Namespace %s
```

Comment:
Do we need to override that?

Reply:
I thought it was nice to override but looking at `systemd-journald@.service` again I can now see that it wasn't necessary, good catch!